src/frontends/ir/tests/rt_info_deserialization.cpp

```diff
@@ -146,43 +131,27 @@ TEST_F(RTInfoDeserialization, NodeV10) {
     auto round = result->get_input_node_ptr(0);
     check_rt_info(round->get_rt_info());
 
-    // read IR v10 with old API
```

Comment:
@olpipi Please revert such code. This code must have to check how legacy API works. Maybe these tests should be moved to ie_func_tests